fea-rs/src/common/glyph_class.rs

```diff
@@ -60,6 +60,9 @@ impl GlyphClass {
 }
 
 impl GlyphSet {
+    /// The empty glyph set
+    pub const EMPTY: Self = GlyphSet(Vec::new());
```

Comment:
GlyphClass has empty a few lines above. It's a bit weird they are inconsistent about this. Perhaps we should standardize on this style since it provides for const?